storage/pipeline/states_replica_update.go

```diff
@@ -187,6 +199,53 @@ func (m *Sealing) handleSubmitReplicaUpdate(ctx statemachine.Context, sector Sec
 	return ctx.Send(SectorReplicaUpdateSubmitted{Message: mcid})
 }
 
+func (m *Sealing) handleWaitMutable(ctx statemachine.Context, sector SectorInfo) error {
+	immutable := true
+	for immutable {
+		ts, err := m.Api.ChainHead(ctx.Context())
+		if err != nil {
+			log.Errorf("handleWaitMutable: api error, not proceeding: %+v", err)
+			return nil
```

Comment:
We probably want to sleep a bit (~60s) and loop around, this can happen when the API is disconnected, which.. can happen

Reply:
Makes a ton of sense.  Becuase I'm just copying this pattern from all over the place inside the fsm I think this issue is out of scope for this PR and should be handled across the board for all fsm states.  Filed #9637 for this